velox/exec/HashBuild.cpp

```diff
@@ -735,10 +735,14 @@ bool HashBuild::finishHashBuild() {
         }
       }
 
-      const bool hasOthers = !otherTables.empty();
+      // TODO: we need to disable parallel join after
```

Comment:
Looks like there are some typos. Would you double check?

Reply:
Updated